### ISSUE TITLE
Project Boom Link Is Dead

### ISSUE THREAD

**Issue by sampoder** [hackclub/site#504]
cc: @kunalbotla 

https://projectboom.org/ is linked on /bank and down, should we use a web archive?

**Comment by kunalbotla** [comment]
working on a fix but there seems to be a dns issue of some kind. should be fixed by monday

**Comment by sarthaktexas** [comment]
FIXED

**Comment by sampoder** [comment]
@sarthaktexas, I still seem to be getting a 404:

```
404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: cle1::7kl2x-1660680270973-d02d2f5b0aef

[legacy.projectboom.org might be available. Click here to learn how to assign it to a project.](https://vercel.com/docs/custom-domains#assigning-a-domain-to-a-git-branch)
```

**Comment by kunalbotla** [comment]
@sampoder can you switch to the most recent web archive for now? i'll change it back to the live one when it's working again

**Comment by sampoder** [comment]
Following up here, is there any alternative site we could use? We've been using the WebArchive one but the CSS is a bit funky on it.

**Comment by tobyab** [comment]
Site seems to be back up and working, we could maybe go back to using [the original link](https://projectboom.org) rather than the web archive.

**Comment by sampoder** [comment]
Amazing, thank you so much for reporting Toby!

**Comment by sampoder** [comment]
Fixed in 78a241d, thank you all!
